### PR TITLE
[docs] Add shortDescription and cleanup repo docs

### PR DESCRIPTION
**Pull request by jesscodez**
This repo needs a bit more work documentation-wise. This is the first pass, which addresses https://github.com/ksonnet/ksonnet/issues/182

### CODE REVIEW

**Comment by jesscodez** [comment]
PTAL @k8k @TamikoT @tehut

File: incubator/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Incubator Registry
+
+## Overview
+
+This directory is an official ksonnet-compatible [registry][2]. If you are unfamiliar with ksonnet, we recommend browsing [the official site][1] to gain more context.
+
+ksonnet's CLI tool (`ks`) is aware of the `incubator` registry out-of-the-box.
```
Comment (tehut) [inline incubator/README.md]: Minor nit but consider (and feel free to reject) changing this to:
Out-of-the-box, ksonnet's CLI tool ('ks') is aware of the 'incubator' registry

File: incubator/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Usage
+
+Assuming that you have the `ks` tool [installed][3], you can use any library in this registry as follows, replacing `redis` with `<library-name>`:
```
Comment (tehut) [inline incubator/README.md]: consider:
..."as follows by replacing 'redis' with...."

**Review by tehut — APPROVED** [review]
I had two tiny, non-blocking and very optional suggestions on the readme but everything looks great overall!

**Comment by jesscodez** [comment]
This works, assuming https://github.com/ksonnet/ksonnet/pull/183 is merged.

Going to force merge since (1) I got Tehut's approval, even though the checker doesn't recognize that 👀  (2) tested locally and confirmed to work

**Pull request merged**